wp-content/civi-extensions/civirazorpay/cli/import-settlement-data.php

```
@@ -0,0 +1,374 @@
+        $returnValues['processed']++;
+      }
+    }
+  }
+
+  /**
+   * Handle retry logic on failure.
+   *
+   * @param Exception $e
+   * @param array $returnValues
+   */
+  private function handleRetry(Exception $e, array &$returnValues): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$returnValues'. (Unused Code Rules)

(UnusedFormalParameter)